piicatcher/explorer/databases.py

```diff
@@ -165,6 +138,8 @@ def factory(cls, ns):
             explorer = MySQLExplorer(ns)
         elif ns.connection_type == "postgres" or ns.connection_type == "redshift":
```

Comment:
This is necessary to get it going:
```suggestion
        elif ns.connection_type == "postgres":
```